src/js/utils/__tests__/TabsUtil-test.js

```diff
@@ -24,22 +25,42 @@ describe('TabsUtil', function () {
       expect(result.length).toEqual(3);
     });
 
-    it('should return li\'s', function () {
+    it('should return LIs', function () {
       var result = TabsUtil.getTabs(this.tabs, 'baz', this.getElement);
 
       expect(result[0].type).toEqual('li');
       expect(result[1].type).toEqual('li');
       expect(result[2].type).toEqual('li');
     });
 
-    it('should return element\'s with one active class', function () {
+    it('should return elements with one active class', function () {
       var result = TabsUtil.getTabs(this.tabs, 'baz', this.getElement);
 
       expect(result[0].props.className).toEqual('tab-item');
       expect(result[1].props.className).toEqual('tab-item active');
       expect(result[2].props.className).toEqual('tab-item');
     });
 
+    it('should not highlight routes contained within the class name',
+        function () {
+          var tabs = TabsUtil.getTabs(this.hierarchicalTabs, '-foo',
```

Comment:
See my note here: https://github.com/dcos/dcos-ui/pull/166/files#r63264866

I think we should update this test to not use the `-` prefix